project-code/app/com/edulify/modules/geolocation/GeolocationModule.java

```diff
@@ -0,0 +1,38 @@
+package com.edulify.modules.geolocation;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import play.Configuration;
+import play.Environment;
+
+/**
+ * Created by sowhat
+ */
+public class GeolocationModule extends AbstractModule {
+
+    private final Environment environment;
```

Comment:
Can try. I was guided by https://playframework.com/documentation/2.4.x/JavaDependencyInjection#Configurable-bindings while implementing this